app/editor/components/EmojiMenu.tsx

```diff
@@ -21,7 +21,7 @@ const searcher = new FuzzySearch<{
   sort: true,
 });
 
-class EmojiMenu extends React.Component<
+class EmojiMenu extends React.PureComponent<
```

Comment:
It takes 10ms to render even when it doesn't render any of the children's 😮
<img width="363" alt="Screenshot 2022-05-01 at 4 27 32 AM" src="https://user-images.githubusercontent.com/40148448/166125299-5f3a339c-4c64-4c24-a8c8-5d84a57173f2.png">